app/models/feeds/megaphone_feed.rb

```diff
@@ -33,10 +42,11 @@ def integration_type
   end
 
   def set_defaults
-    self.slug = "prx-#{id}"
-    self.title = DEFAULT_TITLE
-    self.tokens = [FeedToken.new(label: DEFAULT_TITLE)] if tokens.empty?
+    self.audio_format ||= DEFAULT_AUDIO_FORMAT
     self.private = true
+    self.slug ||= "prx-#{id}"
+    self.title ||= DEFAULT_TITLE
+    self.tokens = [FeedToken.new(label: DEFAULT_TITLE)] if tokens.empty?
```

Comment:
Just to note, this `tokens.empty?` (and the one in the apple feed) do run a SQL query.  I ran into this the other day trying to do an `Episodes.includes(:feeds)` ... I would get an N+1 just from trying to initialize the feeds.

Not right now, but we should probably pick a different hook to init these.

Reply:
oh good point
https://github.com/PRX/feeder.prx.org/issues/1229